src/main/java/racing/Car.java

```diff
@@ -6,24 +6,34 @@
  * @author han
  */
 public class Car {
+
     private final int step;
+    private final String name;
 
-    private Car(int step) {
+    private Car(int step, String name) {
         this.step = step;
+        this.name = name;
     }
 
     public int getStep() {
         return this.step;
     }
 
+    public String getName() {
+        return this.name;
+    }
+
     public static Car create() {
-        return new Car(0);
+        return new Car(0, "temp");
```

Comment:
1. 하나의 주 생성자에 의존하면 어떤 장점이 있을까요?
```suggestion
        return create("temp");
```
2. temp 라는 기본 값을 지정하신 이유가 있으실까요?

Reply:
1. 사용자가 해당 생성자만 이용함으로써, 사용에 있어 생길 수 있는 실수를 막을 수 있지 않을까요
2. 테스트에만 사용될 생성자로 "temp" 라는 기본값을 지정하였네요.